src/lib/builders/context-menu/create.ts

```diff
@@ -216,9 +219,7 @@ export function createContextMenu(props?: CreateContextMenuProps) {
 					/**
 					 * Check for typeahead search and handle it.
 					 */
-					const isCharacterKey = e.key.length === 1;
-					const isModifierKey = e.ctrlKey || e.altKey || e.metaKey;
-					if (!isModifierKey && isCharacterKey) {
+					if (!isModifierKey(e) && (isCharacterKey(e) || isBackspaceKey(e))) {
```

Comment:
seems like we can move this check inside `handleTypeaheadSearch` and not have this repeated in the components.

Reply:
thank you for your feedback! I moved the conditions for the `handleTypeaheadSearch` function to handle

Reply:
that would be great! In case it is helpful, here is a link to the tests I wrote for this functionality for the Radix UI typeahead backspace feature. https://github.com/radix-ui/primitives/blob/92afeb542e539b92763bb36c2e5a3d9773e3bdbf/cypress/integration/Menu.spec.ts